zfs/pool.go

```diff
@@ -68,17 +71,23 @@ func poolNames() ([]string, error) {
 	if err != nil {
 		return nil, err
 	}
+	stderr, err := cmd.StderrPipe()
+	if err != nil {
+		return nil, err
+	}
 	scanner := bufio.NewScanner(out)
 
 	if err = cmd.Start(); err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "Failed to start command %s", cmd.String())
 	}
 
 	for scanner.Scan() {
 		pools = append(pools, scanner.Text())
 	}
+
+	stde, _ := io.ReadAll(stderr)
 	if err = cmd.Wait(); err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, `Failed to wait command %s, err output: "%s"`, cmd.String(), string(stde))
```

Comment:
Replace `errors.Wrapf` with `fmt.Errorf("Failed to execute command '%s'; output: '%s' (%w)", cmd.String(), strings.TrimSpace(string(stde)), err)` to avoid additional dependency, clarify error cause (`wait` is opaque to users), avoid escaped quotes in output, and trim traling newline from output

Reply:
Done too